### DISCUSSION TITLE
Issue with PYMC3 and THEANO

### DISCUSSION THREAD

**Discussion by sleveneu** [pymc-devs/pymc3#6857] [General]
Dear all, 
I've installed PYMC3 and THEANO in an environment. 
When I try to import them from spyder I got 
import pymc3
Traceback (most recent call last):

  Cell In[6], line 1
    import pymc3

  File ~\anaconda3\envs\myenv\lib\site-packages\pymc3\__init__.py:23
    import theano

  File ~\anaconda3\envs\myenv\lib\site-packages\theano\__init__.py:83
    from theano import scalar, tensor

  File ~\anaconda3\envs\myenv\lib\site-packages\theano\scalar\__init__.py:1
    from .basic import *

  File ~\anaconda3\envs\myenv\lib\site-packages\theano\scalar\basic.py:710
    theano.compile.register_view_op_c_code(

AttributeError: partially initialized module 'theano' has no attribute 'compile' (most likely due to a circular import)

import theano
Traceback (most recent call last):

  Cell In[7], line 1
    import theano

  File ~\anaconda3\envs\myenv\lib\site-packages\theano\__init__.py:83
    from theano import scalar, tensor

  File ~\anaconda3\envs\myenv\lib\site-packages\theano\scalar\__init__.py:1
    from .basic import *

  File ~\anaconda3\envs\myenv\lib\site-packages\theano\scalar\basic.py:710
    theano.compile.register_view_op_c_code(

AttributeError: partially initialized module 'theano' has no attribute 'compile' (most likely due to a circular import)

**Comment by twiecki** [comment]
PyMC3 is not supported anymore, you need to install PyMC (e.g. 5.7) which relies on PyTensor: https://www.pymc.io/projects/docs/en/stable/installation.html

**Comment by sleveneu** [comment]
Thanks
